GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -216,6 +230,12 @@ private async void OnContinueClassification(object sender)
                 ClassificationsThisSession += 1;
                 Messenger.Default.Send<int>(ClassificationsThisSession, User);
                 CurrentView = SUMMARY_VIEW;
+
+                if (Notifications.Status == NotificationStatus.HelpingUser)
+                {
+                    Notifications.SendAnswerToUser(SelectedItem);
+                }
+                Notifications.OnResetNotifications(null);
```

Comment:
Seems weird that you are passing `null` here. Do you maybe want to pass `this`?

Reply:
I admit that is rather odd. `onResetNotifications` requires an argument for the `CustomCommand` class. I might be able to set the default param to `null` on the method itself. If not, changing `null` to `this` should do the trick here.